src/Common/Uuid/UuidRegistry.php

```diff
@@ -199,6 +201,31 @@ private static function appendPopulateLog($table, $count, &$logEntry)
         }
     }
 
+    public function createMissingUuidsImproved()
```

Comment:
To convert to this new mechanism, shouldn't this become createMissingUuids() and then the old createMissingUuids() be removed?

Reply:
Wierd... I did this already, I wonder if I did it in a different branch or my rebase messed something up.  I'll change it again.